src/cli/config/update-config-files.ts

```diff
@@ -0,0 +1,24 @@
+import { CliDependencies } from '../cli-dependencies';
+import { updateIgnoredErrorCodes } from './update-ignored-error-codes';
+import { updateLooselyTypeCheckedFilePaths } from './update-loosely-type-checked-file-paths';
+
+const updateConfigFiles = (
+  cliDependencies: CliDependencies,
+  looselyTypeCheckedFilePaths: ReadonlySet<string>,
+  updatedLooselyTypeCheckedFilePaths: Set<string>,
+  ignoredErrorCodesSet: ReadonlySet<string>,
+  updatedIgnoredErrorCodes: Set<string>,
+) => {
+  updateLooselyTypeCheckedFilePaths(
+    cliDependencies,
+    new Set(looselyTypeCheckedFilePaths),
```

Comment:
Minor:

Looks like we should be able to avoid the unnecessary set copying

```suggestion
    looselyTypeCheckedFilePaths,
```